dask/dataframe/io/tests/test_parquet.py

```diff
@@ -369,3 +390,17 @@ def test_timestamp_index():
         ddf.to_parquet(fn)
         ddf2 = dd.read_parquet(fn)
         assert_eq(ddf, ddf2)
+
+
+@pytest.mark.skipif(not pyarrow, reason='pyarrow not found')
+@pytest.mark.xfail(reason="to_parquet does not write nulls by default")
```

Comment:
The default changed in fastparquet, haven't looked at the call in dask yet (since it's been changing).